doc/v2/documentation_data_format.rst

```diff
@@ -95,107 +98,206 @@ problem as such.
-  preequilibration was used, they will override the value obtained from
-  preequilibration. A ``NaN`` value indicates that the original value of the
-  model is to be used (when used in the preequilibration condition, or in the
-  simulation condition if no preequilibration is used) or that the result of
-  preequilibration is to be used (when used in the simulation condition after
-  preequilibration).
-
-  The value in the condition table either replaces the initial value or the
-  value at all timepoints based on whether the model entity has a rate law
-  assigned or not:
-
-  * For model entities that have constant algebraic assignments
-    (but not necessarily constant values), i.e, that do not have a rate of
-    change with respect to time assigned and that are not subject to event
-    assignments, the algebraic assignment is replaced statically at all
-    timepoints. Examples for such model entities are the targets of SBML
-    `AssignmentRules`.
-
-  * For all other entities, e.g., those that are assigned by SBML `RateRules`,
-    only the initial value can be assigned in the condition table. If an
-    assignment of the rate of change with respect to time or event assignment
-    is desired, the values of model entities that are used to define rate of
-    change or event assignments must be assigned in the condition table.
-    If no such model entities exist, assignment is not possible.
+- ``targetId``
+  [NON_ESTIMATED_ENTITY_ID, required except for ``operationType=noChange``]
 
+  The ID of the non-estimated entity that will change.
+  Different restrictions apply depending on the ``operationType`` and the
+  type of the model.
+
+- ``operationType`` [STRING, REQUIRED]
+
+  How the target value is to be interpreted. Allowed values are:
+
+  - ``setCurrentValue``: The current value of the target is initialized to the
+    value of ``targetValue`` when the respective condition is applied.
+    The target must be a constant target or a differential target.
+
+    Any symbols in ``targetValue`` are interpreted as their value at the time
+    just before this change is applied (similar
+    to an SBML event with ``useValuesFromTriggerTime=True``; in case of the
+    first period of a timecourse, they will be interpreted as their initial
+    values as defined in the model).
+
+    **TODO** update
+
+  - ``noChange``: This value is used to indicate that no change is associated
+    with this condition. This is useful in cases where an explicit condition
+    ID is required, but no change is to be applied. (E.g., if pre-simulation
+    up to a steady state is to be performed using the default values of the
+    model, followed by a different condition.)
+    ``targetId`` and ``targetValue`` have to be empty in this case.
+    For any ``conditionId`` with ``operationType=noChange``, there must not be
+    any other row with the same ``conditionId`` in the conditions table.
```

Comment:
@FFroehlich suggested to completely remove `operationType` because after the recent simplification.  

`noChange` would then be specified by an empty conditionID in the experiments table.

Works for me.

@PEtab-dev/petab-editors : What are your thoughts?